test/webhook-manager.js

```diff
@@ -0,0 +1,146 @@
+const request = require('supertest')
+const ngrok = require('ngrok')
+
+const { getAccessTokenHeaders } = require('./auth')
+
+const noop = () => Promise.resolve()
+
+// Utility to test external service webhooks integration into Stelace
+// Two modes of functioning:
+// - test webhooks for real via Ngrok
+// - simulate webhooks by fetching and creating events
+//    (suitable if webhooks cannot be created via API and one cannot have access to Ngrok paid plan
+//     for reserved subdomains)
```

Comment:
I would be more precise and make the distinction between external events and Stelace events:

```js
// Two operating modes:
// - Test external webhook integration for real via Ngrok,
//     as long as the external service lets you create webhooks with some API,
//     or you have a paid ngrok plan and a stable URL to use for a longlived test webhook
// - Simulate external webhook by polling external service events
//    and propagate them to some Stelace endpoint, provided the external service
//    has some Event API to fetch latest external events.
```